packages/components/src/local/atoms/game-title/types/props.d.ts

```diff
@@ -5,4 +5,8 @@ export default interface Props extends WargameList {
    * Callback on clicking the game title
    */
   onClick?: (title: string) => void
+  /**
+   * Container element ref
+   */
+   ref?: React.Ref<any>
```

Comment:
Could you try to give the element a `type`, please?  I presume it's a dom-element.

Reply:
Let me check

Reply:
Done